### ISSUE TITLE
Delete files using Storage

### ISSUE THREAD

**Issue by helderferrari2** [nao-pon/flysystem-google-drive#79]
Hello guys,

I wanna delete some files by filename or fullpath, is possible using STORAGE?

$arrFilenames = ['file1.png', 'file2.png'];
Storage::disk('google')->delete($arrFilenames);

**Comment by DeeSouza** [comment]
Do you have found a solution for this issue?

**Comment by helderferrari2** [comment]
@DeeSouza No, I don't, to be honest, I just remove the filepath from my database.

**Comment by nmfzone** [comment]
You can't delete file using your original file name / path. You need to delete those file using Google Drive specific file name / path. You can get Google Drive specific path when you execute `->write()` or `->writeStream()` function, it would return an array, the `path` key is your file path in Google Drive, it's something like

`1AcrcenV7XzIQQDogsURDF2n2WsBBBoPj/1anD6QH3peVXbCvuJ693viCOc7xLjOaJ9`.

**Comment by PaolaRuby** [comment]
@DeeSouza did you try [flysystem-google-drive-ext](https://github.com/masbug/flysystem-google-drive-ext) , it can delete files only with the path
